app/models/concerns/waste_carriers_engine/can_change_registration_status.rb

```diff
@@ -78,82 +83,34 @@ def log_status_change
     # Guards
     def renewal_allowed?
       return true if renewal_application_submitted?
```

Comment:
I think we can skip the `renewal_application_submitted?` check now, as nothing should make it to `renewal_received` without the declaration being accepted.

However we do use that method for other things, so it should continue to exist - just not be used here.